classes/Tools.php

```diff
@@ -42,6 +42,10 @@ class ToolsCore
     const SERVICE_LOCALE_REPOSITORY = 'prestashop.core.localization.locale.repository';
     public const CACHE_LIFETIME_SECONDS = 604800;
 
+    public const DEFAULT_CONNECTION_TIMEOUT = 5;
+    public const DEFAULT_ADDONS_TIMEOUT = 30;
```

Comment:
Don't forget to update this part:
![image](https://user-images.githubusercontent.com/1462701/125291796-16739980-e322-11eb-91d0-09c6d65c882e.png)

Reply:
I change it directly in the `ApiClient` class via `setTimeout/setConnectionTimeout`